openwisp_monitoring/device/static/monitoring/js/chart.css

```
@@ -0,0 +1,156 @@
+  background: url(../../admin/img/icon-unknown.svg) 0 0 no-repeat;
+  background-size: 80%;
+  background-position: 50% 50%;
+  opacity: 0.6;
+  cursor: pointer;
+  margin-left: 3px;
+}
+.chart-heading {
+  margin: 0;
+  font-size: 2em;
+  text-align: center;
+  font-size: 1.22em;
```

Biome (2.4.6)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateProperties)